autosklearn/experimental/askl2.py

```diff
@@ -272,7 +273,7 @@ def __init__(
             If None is provided, a default metric is selected depending on the task.
 
         scoring_functions : List[Scorer], optional (None)
-            List of scorers which will be calculated for each pipeline and results will be 
+            List of scorers which will be calculated for each pipeline and results will be
```

Comment:
Shouldn't PEP8 always be checked on all files? Could you please check if it is somehow not enabled?

Reply:
flake8 W291 only applies to file outside of comments.

My editor is setup to fix it everywhere. If I intentionally leave a white space on a non-comment line, pre-commit fails.